Libraries/Network/FormData.js

```diff
@@ -85,7 +81,7 @@ class FormData {
         }
         return {...value, headers, fieldName: name};
       }
-      // Cast to string all other values
+      // Convert non-object values to strings as per FormData.append() spec
       return {string: String(value), headers, fieldName: name};
```

Comment:
We should probably remove this extra conversion instead of removing the one at the top